src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -95,14 +92,14 @@ public void periodic() {
                         m_rearRight.getPosition()
                 });
 
-        //Logger logger = Logger.getInstance();
-        //logger.recordOutput("Chassis/Actual", new SwerveModuleState[] {
-          //      m_frontLeft.getState(),
-            //    m_frontRight.getState(),
-              //  m_rearLeft.getState(),
-                //m_rearRight.getState()
-    //    });
-      //  logger.recordOutput("Chassis/Pose", getPose());
+        Logger logger = Logger.getInstance();
+        logger.recordOutput("Chassis/Actual", new SwerveModuleState[] {
+                m_frontLeft.getState(),
+                m_frontRight.getState(),
+                m_rearLeft.getState(),
+                m_rearRight.getState()
+        });
+        logger.recordOutput("Chassis/Pose", getPose());
```

Comment:
Instead of using Logger.getInstance then calling stuff on the logger we can directly call the3se functions on the Logger class, ex:
```
Logger.recordOutput
```